build.sbt

```diff
@@ -31,6 +31,9 @@ libraryDependencies ++= Seq(
     exclude("org.bouncycastle", "bcprov-ext-jdk15on")
     exclude("org.bouncycastle", "bcutil-jdk15on")
     exclude("org.bouncycastle", "bcpkix-jdk15on"),
+  "org.broadinstitute.dsde.workbench" %% "workbench-notifications" % "0.3-3086a4df-SNAP"
```

Comment:
Will update to a non-SNAP version once https://github.com/broadinstitute/workbench-libs/pull/1054 is merged